packages/valory/skills/hello_world_abci/tests/test_payloads.py

```diff
@@ -76,12 +74,12 @@ def test_print_message_payload() -> None:
 def test_reset_payload() -> None:
     """Test `ResetPayload`"""
 
-    payload = ResetPayload(sender="sender", period_count=1, id_="id")
+    payload = ResetPayload(sender="sender", period_count=1)
 
     assert payload.period_count == 1
-    assert payload.id_ == "id"
+    assert payload.id_
     assert payload.data == {"period_count": 1}
-    assert hash(payload) == hash(tuple(sorted(payload.data.items())))
+    assert hash(payload)
```

Comment:
no more custom `__hash__` method present. Should remain hashable though since the data structure now is immutable, and that is what we care about